testutils/base.go

```diff
@@ -24,22 +24,20 @@ import (
 	"github.com/cockroachdb/cockroach/security"
 )
 
-// NewTestBaseContext creates a base context for testing.
+// NewRootTestBaseContext creates a base context for testing.
 // This uses embedded certs and the "root" user (default client user).
-// The certs file loader is overriden in individual main_test files.
-// This is meant to be used by external clients (as opposed to nodes).
-func NewTestBaseContext() *base.Context {
+// The "root" user has client certificates only.
+func NewRootTestBaseContext() *base.Context {
```

Comment:
Another thought: the `base.Context` appears to be unchanged. There isn't much advantage to a method right now. Why not just have `RootTestBaseContext` and `NodeTestBaseContext` vars defined here? Also, we should probably be passing around `base.Context` everywhere instead of `*base.Context`.